client/my-sites/plugins/style.scss

```diff
@@ -160,6 +161,13 @@ $plugin-details-header-padding: 100px;
 	}
 }
 
+%placeholder {
+	cursor: default;
+	color: transparent;
+	background-color: var( --color-neutral-0 );
+	animation: loading-fade 1.6s ease-in-out infinite;
+}
+
```

Comment:
TIL about the placeholder selector
https://sass-lang.com/documentation/style-rules/placeholder-selectors

Reply:
+1

Reply:
+1 while working on this! 😄 I didn't want to copy that same CSS into two rules, and I knew that Sass has some mechanism for this use case.